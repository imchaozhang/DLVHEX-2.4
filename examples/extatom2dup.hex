prefix("testa:").
prefix("testb:").
more("a","b","c").
more("d","e","f").
limit("testa:","testa: a").
limit("testb:","testb: d").
complete(F) :- more(X,Y,Z),prefix(S),
                  &testConcat[S," "](A),
                  &testConcat[A,X](B),
                  &testConcat[B," "](C),
                  &testConcat[C,Y](D),
                  &testConcat[D," "](E),
                  &testConcat[E,Z](F),
									limit(S,B).
