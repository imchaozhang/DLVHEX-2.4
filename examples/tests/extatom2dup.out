{complete("testa: a b c"),complete("testb: d e f")}
